nft/nfts.yaml

```diff
@@ -876,14 +889,14 @@ components:
       required: false
     startLogIndex:
       name: startLogIndex
-      description: Integer - The log index within the startBlock to start fetching NFT sales data from.
+      description: Integer - The log index within the startBlock to start fetching NFT sales data from. Defaults to 0.
```

Comment:
Ah yep I can add this but it currently slows the query down since we filter by log index only if it is provided in a query param, will case for 0 so we don't do this.